.github/workflows/test.yml

```diff
@@ -0,0 +1,33 @@
+name: CI/CD for Nest.js
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
```

Comment:
I am sorry I mentioned 
> update to actions/setup-node@v4

but the latest version is [3](https://github.com/actions/setup-node/tree/main). So please change it to 3

Reply:
done changed it to 3